performance-tests/performance.test.js

```diff
@@ -105,7 +105,7 @@ describe('Tram-One - Performance Tests', () => {
 		// what does quickly mean? 5x elements should not be more than 5x slower
 		// we'll use the average, since that should be the least error prone (with the outliers removed)
 		// we'll also add a buffer to the right, to account for fragility
-		const buffer = 1 // 1 seconds, which will map to 5 seconds on the left
+		const buffer = 2 // 2 seconds, which will map to 10 seconds on the left
```

Comment:
this test was failing intermittently on CircleCI, so this should fix it (in reality, a difference of 10 seconds is more than fine.

Reply:
I guess, it does matter for the small cases though, maybe it should be a percentage of the average time :thinking: